### PR TITLE
Branding_console add ansible playbook links 

### PR DESCRIPTION
**Pull request by Jooho**
#### What is this PR About?
After ansible playbooks are merged to openshift-toolkit, the branding_console doc need to add those links.

#### How should we test or review this PR?
This is simple link update so I believe that clicking those links are enough to review

#### Is there a relevant Trello card or Github issue open for this?
Nope

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

### CODE REVIEW

File: playbooks/operationalizing/branding_console.adoc [hunk 1]
```diff
@@ -273,17 +273,12 @@ Use the css class in a template::
 <1> This value must be same as the css style in extension.css
 
 []
-== Ansible Playbook
-This ansible playbook contains 4 roles : resize_image, configure_login_logo, configure_webconsole_logo, demo_template_icon_change.
-Those roles can be executed by 1 playbook or each role can be executed. This chapter will explain how to configure "Branding console" with ansible playbook.
+== Ansible Playbooks
+Using below ansible playbook, it can be easy to change logo of OpenShift Login Page and Web Console. 
 
+*Playbook Lists*
```
Comment (etsauer) [inline playbooks/operationalizing/branding_console.adoc]: @Jooho should this say `Playbooks Links`?

File: playbooks/operationalizing/branding_console.adoc [hunk 2]
```diff
@@ -273,17 +273,12 @@ Use the css class in a template::
...
 
-=== Ansible role
-- resize_image
-- configure_login_logo
-- configure_webconsole_logo
-- demo_template_icon_change
+- https://github.com/redhat-cop/openshift-toolkit/tree/master/ansible-playbook-openshift-custom-login-page[ansible-playbook-openshift-custom-login-page]
```
Comment (etsauer) [inline playbooks/operationalizing/branding_console.adoc]: need to preface with `link:https...`

File: playbooks/operationalizing/branding_console.adoc (hunk 1 above)
Comment (Jooho) [reply playbooks/operationalizing/branding_console.adoc]: ^^ I don't mind..I will update it

File: playbooks/operationalizing/branding_console.adoc (hunk 2 above)
Comment (Jooho) [reply playbooks/operationalizing/branding_console.adoc]: OK, it is my bad.. I will refer another doc. then will update it.

**Comment by Jooho** [comment]
@etsauer Updated ^^

**Pull request merged**